### PR TITLE
DS-3583 Usage of correct Collection Array

### PR DESCRIPTION
**Pull request by jonas-atmire**
Jira Ticket
https://jira.duraspace.org/browse/DS-3583

### CODE REVIEW

File: dspace-api/src/main/java/org/dspace/app/itemimport/ItemImport.java [hunk 1]
```diff
@@ -66,6 +42,19 @@
...
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.TransformerException;
+import java.io.*;
+import java.net.URL;
+import java.sql.SQLException;
+import java.text.SimpleDateFormat;
+import java.util.*;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+
```
Comment (tdonohue) [inline dspace-api/src/main/java/org/dspace/app/itemimport/ItemImport.java]: All of the moving of `import` statements here is unnecessary on a maintenance branch (5.x), as it won't be brought forward into master or other branches. I'd rather we just update this to be a one-liner, as that means it better aligns with #1241 (which is the same fix for 6.x).

**Comment by mwoodiupui** [comment]
+1 obviously correct.  I agree with tdonohue's comment concerning the rearrangement of imports.

**Comment by jonas-atmire** [comment]
I hadn't noticed that my development environment automatically reordered the imports, I completely agree that this isn't part of the PR.
Reverted back to original arrangement and made sure only the appropriate line is changed

**Review by tdonohue — APPROVED** [review]
👍 Looks good now, thanks @jonas-atmire!

**Pull request merged**